storybook/src/Storybook/PageToolBar.qml

```diff
@@ -25,6 +28,29 @@ ToolBar {
             background: null
         }
 
+        ToolButton {
```

Comment:
Why not add the same kbd shortcut too? Just an idea

Reply:
Could you elaborate?

Reply:
Nvm, I thought "Ctrl+K, Ctrl+V" copies the name of the component in QtCreator, so I thought why not add it here as well :)